src/components/table/examples/table-sortable/index.njk

```diff
@@ -68,16 +68,15 @@
                             "dataSort": "0"
                         },
                         {
-                            "form": {
-                                "method": "Get",
-                                "action": "#" 
-                                "button": {
-                                    "type": "button",
-                                    "text": "View details",
-                                    "classes": "ons-btn--secondary",
-                                    "url": "#0"
-                                }
-                            }
+                            "value": onsButton({
```

Comment:
It does work, see example: https://github.com/ONSdigital/spp-reference-data/blob/dd45c7dd992d222ffac0c1dc9374283496220aa9/lambdas/spp-reference-data/spp_reference_data/templates/dataset_list_view.html#L42

Reply:
The build has failed https://github.com/ONSdigital/design-system/pull/2211/checks#step:5:37

It could be that it works when used inside a variable that gets appended but it doesn't seem to be working when using directly inside the table macro config.